lectures/08-a/theory.tex

```diff
@@ -0,0 +1,262 @@
+\chapter{Energy Based Models}
+% Authors: Diksha Meghwal, Chiao-Hsun Wang, Imran
+% Lecture date: 04/01/2019
+
+\section{Recap}
+% Authors: Diksha Meghwal
+% Lecture date: 04/01/2019
+
+A quick recap, energy based models(EBMs) are functions that take an input and product a scalar output as a result of some optimization. Eg:
+
+\begin{figure}[htb]
+    \centering
+    \begin{subfigure}[b]{0.25\textwidth}
+        \includegraphics[width=\textwidth]{lectures/08-a/images/conditional_EBM.png}
+        \caption{Conditional model}
+        \label{fig:conditional}
```

Comment:
This is already defined in `lectures/07-b`. Change it.

Reply:
Hi Alfred,
It seems like you've applied the change,
do we have to add anything else?
Thanks!